xtra-bitmex-price-feed/src/lib.rs

```diff
@@ -13,17 +13,18 @@ use xtra_productivity::xtra_productivity;
 pub const QUOTE_INTERVAL_MINUTES: i64 = 1;
 
 pub struct Actor {
-    latest_quote: HashMap<ContractSymbol, Quote>,
+    latest_quotes: LatestQuotes,
 
     /// Contains the reason we are stopping.
     stop_reason: Option<Error>,
     network: Network,
 }
 
+/// Subscribes to BitMEX and retrieves latest quotes for BTCUSD and ETHUSD.
```

Comment:
🙃 I would expect this to go on the `struct`.

Reply:
lol, indeed! :)